src/DockerContainer.php

```diff
@@ -86,6 +91,13 @@ public function mapPort(int $portOnHost, $portOnDocker): self
         return $this;
     }
 
+    public function setVariable(string $envName, string $envValue): self
```

Comment:
Let's make it very explicit and rename this to `setEnvironmentVariable`

Reply:
That's fine by me! I did think about calling it that.